gen/definitions/profile_parcels/service_lan_vpn.yaml

```diff
@@ -0,0 +1,423 @@
+---
+name: Service LAN VPN
+rest_endpoint: /v1/feature-profile/sdwan/service/%v/lan/vpn
+minimum_version: 20.12.0
+test_tags: [SDWAN_2012]
+attributes:
+  - tf_name: feature_profile_id
+    reference: true
+    type: String
+    mandatory: true
+    description: Feature Profile ID
+    example: f6dd22c8-0b4f-496c-9a0b-6813d1f8b8ac
+    test_value: sdwan_service_feature_profile.test.id
+  - model_name: vpnId
+    tf_name: vpn
+    example: 1
+  - model_name: name
+    tf_name: config_description
+    example: VPN1
+  - model_name: ompAdminDistance
+    tf_name: omp_admin_distance_ipv4
+    example: 1
+  - model_name: ompAdminDistanceIpv6
+    tf_name: omp_admin_distance_ipv6
+    example: 1
+  - model_name: enableSdra
+    tf_name: enable_sdwan_remote_access
+    example: false
+  - model_name: primaryDnsAddressIpv4
+    data_path: [dnsIpv4]
+    mandatory: true
```

Comment:
Is this really mandatory or can it be an ipv6 dns server only as well?

Reply:
Yes, when testing with exclude_null it returns, {"Required But Missing Attributes":["data.dnsIpv4.primaryDnsAddressIpv4","data.dnsIpv6.primaryDnsAddressIpv6"] and when testing with only primaryDnsAddressIpv6, return {"Required But Missing Attributes\":["data.dnsIpv4.primaryDnsAddressIpv4"]}